pkg/util/util.go

```diff
@@ -249,3 +255,82 @@ func OpenBrowser(url string) error {
 
 	return nil
 }
+
+// IsDebug returns true if we are debugging (-v is set to anything but 0)
+func IsDebug() bool {
+
+	flag := pflag.Lookup("v")
+
+	if flag != nil {
+		return !strings.Contains(pflag.Lookup("v").Value.String(), "0")
+	}
+
+	return false
+}
+
+// LogNamef will output the name of the component / application / project in a *bolded* / underlined manner
+func LogNamef(format string, a ...interface{}) {
```

Comment:
would it make sense to put all `Log*` functions into separate file in this package? something like `logging.go`